CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ These are the section headers that we use:
 
 ## [Unreleased]
 
+### Added
+
+- Added `RankingQuestionSettings` class allowing to create ranking questions in the API ([#3232](https://github.com/argilla-io/argilla/pull/3232))
```

Comment:
Maybe make direct reference to the API endpoint  `/api/v1/datasatet/:dataset_id/questions` now support RankingQUestionSettings, bla blablab